src/main/java/org/jabref/logic/exporter/ExporterFactory.java

```diff
@@ -27,6 +30,7 @@ public static ExporterFactory create(CliPreferences preferences) {
         List<TemplateExporter> customFormats = preferences.getExportPreferences().getCustomExporters();
         LayoutFormatterPreferences layoutPreferences = preferences.getLayoutFormatterPreferences();
         SelfContainedSaveOrder saveOrder = SelfContainedSaveOrder.of(preferences.getSelfContainedExportConfiguration().getSaveOrder());
+        SelfContainedSaveOrder saveMostRecentFirst = new SelfContainedSaveOrder(SaveOrder.OrderType.SPECIFIED, List.of(new SortCriterion(StandardField.YEAR, true)));
```

Comment:
I would simply use the save order from the preferences, just for consistency and to reduce wtfs ;)